aws/logs_monitoring/lambda_function.py

```diff
@@ -309,19 +322,41 @@ def scrub(self, payload):
         return payload
 
 
-def lambda_handler(event, context):
+def lambda_handler_wrapped(event, context):
```

Comment:
Can we use a more explicit name here please ? I don't think `wrapped` is very explicit

Reply:
Renamed it to `datadog_log_forwarder`.